src/admin/permission-groups/permission-group.service.ts

```diff
@@ -28,15 +28,15 @@ export class PermissionGroupService {
 		page: number,
 		sortColumn: PermissionGroupOverviewTableCols,
 		sortOrder: Avo.Search.OrderDirection,
-		queryText: string
+		where: any
 	): Promise<[PermissionGroup[], number]> {
 		let variables: any;
 		try {
 			variables = {
+				where,
```

Comment:
In hoeverre kan hier `whitelist`-enforcing doorgevoerd worden, als in mensen kunnen nu willekeurig welke where meesturen, toch?

Reply:
klopt, maar hier is het moeilijk om te vormen naar afzonderlijke props aangezien die dynamisch het _and filter object opbouwt afhakelijk van hoeveel woorden er in je zoek query staan.
Zie deze regels: https://github.com/viaacode/avo2-client/pull/469/files#diff-83c99d8feac32e739aef2759e1468dceR50-R55

De hele permissions tabel zal nog afgesloten worden voor iedereen tenzij de server en users met de admin role via deze methode: https://district01.atlassian.net/browse/AVO2-928